eng/Directory.Build.Data.props

```diff
@@ -159,6 +159,10 @@
     <ServerGarbageCollection>true</ServerGarbageCollection>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(IsPerfProject)' == 'true' or '$(IsStressProject)' == 'true'">
+    <PerfStressSharedSources>$(MSBuildThisFileDirectory)/../common/PerfStressShared/</PerfStressSharedSources>
```

Comment:
Does PerfStressShared contain the code that service stress tests would use? If not we shouldn't put it into the global props.

Reply:
No, it's only shared by the perf and stress frameworks.  There is currently no MSBuild property for the frameworks but not the service tests.  Should I just use relative path in the framework csproj instead?

Reply:
Yes. I would like to prefer not to overload the global props file when possible.